StyleCop.Analyzers/StyleCop.Analyzers/Settings/SettingsHelper.cs

```diff
@@ -159,24 +178,28 @@ private static StyleCopSettings GetStyleCopSettings(string path, SourceText text
             return new StyleCopSettings();
         }
 
-        private static SourceText TryGetStyleCopSettingsText(this AnalyzerOptions options, CancellationToken cancellationToken)
+        private static SourceText TryGetStyleCopSettingsText(this AnalyzerOptions options, CancellationToken cancellationToken, out string settingsFileName)
```

Comment:
:bulb: This is a file path, not a file name. It would be good to reflect this in the parameter name.

Reply:
Good catch, the parameter name is changed.